### ISSUE TITLE
If the player choses an invalid square, no action is taken.

### ISSUE THREAD

**Issue by paolovic** [sopra-fs22-group-16/sopra-fs22-hannibal-server#74]

**Comment by paolovic** [comment]
shouldn't the game display some error message in that case? @apple00juice @luis-tm @mkorob

**Comment by apple00juice** [comment]
I don't think so, maybe you just want to see where the unit could walk, or what the unit could attack. For this you need a way to cancel the move/attack, clicking outside of the possible options is a easy and I think intuitive way to do that.

**Comment by paolovic** [comment]
ah ok
